src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -5,7 +5,7 @@
     <!--
     TODO: Disable this exception, and actually do document all public API.
     -->
-    <NoWarn>$(NoWarn),1591</NoWarn>
+    <NoWarn>$(NoWarn),1591,CS0618</NoWarn>
```

Comment:
Will open a new issue to move CS0618 nowarn to individual metric related files, not for whole project.

Reply:
@Austin-Tan Can you work on this? (a separate PR)

Reply:
Sure I'll get started right now. Would you recommend just Ctrl+F'ing the project for references to components in src/OpenTelemetry/Metrics, and appending `#pragma warning disable cs0618` to the top?
And double checking my understanding - why don't we want compiler warnings? Aren't we intending to make it clear that Metrics are experimental?

Reply:
Metrics are marked Obsolete so customers should be alerted. But there are pieces in this repo itself which depends on the metric, so they should be excluded from this warning.

(current its a project level warning exclusion, which will prevent us from catching any real issues, outside of metrics).